### PR TITLE
Prevent returning null on stageTxV2 mutation

### PR DESCRIPTION
**Pull request by BasixKOR**
This prevents InvalidOperationException which occurs when trying to return null value on NonNullGraphType.

## Stack trace when returned null
```
System.InvalidOperationException: Cannot return null for a non-null type. Field: stageTxV2, Type: TxId!.
   at GraphQL.Execution.ExecutionStrategy.ValidateNodeResult(ExecutionContext context, ExecutionNode node) in /_/src/GraphQL/Execution/ExecutionStrategy.cs:line 566
   at GraphQL.Execution.ExecutionStrategy.CompleteNode(ExecutionContext context, ExecutionNode node) in /_/src/GraphQL/Execution/ExecutionStrategy.cs:line 480
```

## Reference
 - ["Input errors", GraphQL.NET (Accessed 2022.02.25 KST)](https://graphql-dotnet.github.io/docs/getting-started/errors/#input-errors)

### CODE REVIEW

**Review by ipdae — APPROVED** [review]

**Review by moreal — APPROVED** [review]

**Comment by moreal** [comment]
I resolved the conflict. After the build succeeds, I'll merge this pull request because it has enough approvals.

**Pull request merged**
